glom/core.py

```diff
@@ -112,6 +113,8 @@
 
 MODE =  make_sentinel('MODE')
 
+MIN_MODE =  make_sentinel('MIN_MODE')
```

Comment:
Do we describe minor modes anywhere? (comments/docs)

Reply:
that's a pretty big oversight, good point

I think there may be some docstrings attached to ARG / arg_val